ServerCore/Pages/Resources/pd2021.2203/HomePartial.cshtml

```diff
@@ -0,0 +1,60 @@
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    bool IsRegistered = false;
+    if (eventPage != null)
+    {
+        IsRegistered = await eventPage.IsRegisteredUser();
+    }
+}
+
+<div class="container" id="pd2021-container">
+    <h2 class="pd2021-title">
+        <span>Intern Puzzleday 2021 ver. 2022</span>
+    </h2>
+    <div>
+        <img class="divider-img" src="~/images/divider.png">
+    </div>
+
+    <div class="pd2021-container-root">
+        <div class="pd2021-container-card">
+            <h3>WELCOME TO PUZZLEDAY</h3>
+            <p>We&rsquo;re honored to have you join us in what we&rsquo;re sure will be an event for the history books: Intern Puzzleday 2021: 2022 Edition! Our authors have written enough puzzles to keep your eyes glued to the pages for two days this March. And the best news: you don&rsquo;t need to worry about late fees!</p>
+
+            <p>This year&rsquo;s event will take place over two days: <b>Thursday, March 24<sup>th</sup></b> and <b>Friday, March 25<sup>th</sup></b>. On both of those days, you&rsquo;ll have 4 hours from <b>1pm-5pm UTC</b> to solve puzzles, and both days of solving contribute to the overall event.  Some may also choose to keep solving after the end of the first day, but doing so is not required.</p>
+
+            <p>Take a look at our <b><a asp-page="/EventSpecific/FAQ">FAQ</a></b> and <b><a asp-page="/EventSpecific/Rules">Rules</a></b> if you have any questions.</p>
+
+            <p>We invite participation by FTEs, friends, and families of all abilities. If you require an accommodation,  please reach out to us at <a href="mailto:puzzhunt@microsoft.com">puzzhunt@microsoft.com</a>.</p>
+        </div>
+
+        <div class="pd2021-container-card">
+            <h3>SCHEDULE (all times UTC)</h3>
+
+            <h4>PREGAME PUZZLES (optional)</h4>
+
+            <p>These are a great way to get into the spirit of the event and learn how to use our website.  We hope they are fun and give you an idea of what to expect.  The puzzles will be released in the days leading up to the event, and can be found here by clicking &quot;Puzzles&quot; at the top of the page.  If you don't see a link to the puzzles up there, make sure you're registered to play!</p>
+
+            <p>These puzzles are primarily for practice, and we&rsquo;ll go over the solutions at the opening session.</p>
+
+            <h4>EVENT SCHEDULE</h4>
+
+            <p>On <b>March 24<sup>th</sup></b> at <b>1pm</b>, there will be a short opening session where we will go over rules and theme. After that, your team will have until <b>5pm</b> to solve as many puzzles as you can! We&rsquo;ll take a snapshot of the leaderboard at that time, but these puzzles will still be available to solve until the end of the second day. <!--<ul><li><b><a href="https://aka.ms/nipdopening" target="_blank">Click here to join the opening session!</a></b> (Will be live shortly before scheduled start time)</li></ul>--></p>
```

Comment:
Do you need to say UTC?

Reply:
In the surrounding block there's a disclaimer that all times are UTC since we'll be adding info about the opening and closing there, and that way we don't need to worry about forgetting to mention the time zone every time.